src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -256,8 +256,21 @@ private static IncrementalMethodStubGenerationContext CalculateStubInformation(M
                 generatorDiagnostics.ReportConfigurationNotSupported(lcidConversionAttr, nameof(TypeNames.LCIDConversionAttribute));
             }
 
+            var args = generatedComAttribute?.NamedArguments.ToImmutableDictionary();
+            var iacd = new InteropAttributeCompilationData();
+            if (args is not null)
+                iacd = iacd.WithValuesFromNamedArguments(args);
```

Comment:
This is a good example of where the `?` is doing a lot of lifting and more harm than good. Please break this apart and branch on `generatedComAttribute` being non-`null` and then update. There is no need for `args` to be `null` in this case, but it is because of the use of `?`.

Reply:
That means `var iacd = new InteropAttributeCompilationData();` is done first.